src/components/breadcrumb/breadcrumb.js

```diff
@@ -15,16 +16,16 @@ class Breadcrumb extends React.Component {
   render() {
     const { t, threadName, threadId, totalTasks, taskPosition } = this.props;
     return (
-      <PfBreadcrumb className="integr8ly-breadcrumb pf-u-pl-lg">
```

Comment:
You can delete this `integr8ly-breadcrumb`, as we're doing a 'reset' to default PatternFly 4 styles on converted components.